receiver.py

```diff
@@ -72,13 +82,25 @@ def data_lines(self):
                         'source_addr_long', self.source_addr_long)
                 self.source_addr = packet.get(
                         'source_addr', self.source_addr)
+                self.rssi = packet.get('rssi', self.source_addr)
 
                 payload += packet['rf_data']
-                if x < self.expected_packets - 1 and len(payload) < 100:
-                    break;
 
-            # let our data be processed
-            yield self.data_shape.unpack(payload)
+                # Read first two bytes, to determine packet type
```

Comment:
This change in indent is not correct. it will happen to work, because you only expect telemetry to occupy a single packet, but the preceding for-loop in xrange(expected_packets) should be what handles splitting telemetry lines across several packets.